serialization.md

```diff
@@ -0,0 +1,43 @@
+# CloudEvents Serialization Profile - Version 0.1
+
+This document specified how a CloudEvent is to be serialized into certain
+encoding formats. Compliant CloudEvents implementations that support these
+formats MUST adhere to these rules for these formats.
+
+## Table of Contents
+- [Notational Conventions](#notational-conventions)
+- [JSON](#json)
+
+## Notational Conventions
+
+The key words "MUST", "MUST NOT", "REQUIRED", "SHALL", "SHALL NOT", "SHOULD",
+"SHOULD NOT", "RECOMMENDED", "MAY", and "OPTIONAL" in this document are to
+be interpreted as described in [RFC 2119](https://tools.ietf.org/html/rfc2119).
+
+## JSON
+
+When serialized in [JSON](https://tools.ietf.org/html/rfc7159) a CloudEvent
+MUST adhere to the following format:
+
+```
+{
+  "cloud-events-version": "cloud-events-version value",
+  "event-id": "event-id value",
+  "namespace": "namespace value",
+  "source": {
+    "type": "source-id value",
+    "id": "source-id value"
+  },
+  "event-type": "event-type value",
+  "event-type-version": "event-type value",
+  "schema-url": "schema-url value",
+  "extensions": {
+    ... extensions values ...
+  },
+  "data": ... data value ...
+}
+```
+
+Notes:
```

Comment:
Should there be a comment about extra fields in the serialized data? Such as:
```
- It is undefined whether extra fields are passed along or removed depending on the usage role and implementation. Those fields are best placed in the extensions map.
```

Reply:
I expanded on the thought a little - see what you think.

Reply:
Thanks for the update. LGTM.